coala_quickstart/coala_quickstart.py

```diff
@@ -57,13 +58,18 @@ def _get_arg_parser():
     return arg_parser
 
 
+def signal_handler(signal, frame):
```

Comment:
What is the need of passing arguments to this function if you are not using them?

Reply:
I get the following error if i don't pass the arguments,
TypeError: signal_handler() takes 0 positional arguments but 2 were given

Reply:
@jackthepirate6 thats probably because you are forgetting `self` or `cls`. You need to remove those unnecessary arguments.